### ISSUE TITLE
ARL window doesnt show on 2nd access.

### ISSUE THREAD

**Issue by Ripper007** [Sweepi/ARL_WotLK#1]
With the Ulduar patch the addon works only one time - scan results are shown - but starts bugging out after, trying to click it 2nd time doesn't pop the window anymore. Relogging fix it temporary but bug repeats. It seems that only cooking shows this problem.

**Comment by gravitas425** [comment]
Alchemy does the same. 
Info from bugsack:
772x AckisRecipeList/List.lua:1509: Usage: self:SetFont(fontFile, height, flags)
[string "=[C]"]: in function `SetFont'
[string "@AckisRecipeList/List.lua"]:1509: in function <AckisRecipeList/List.lua:1491>
[string "@AckisRecipeList/List.lua"]:1880: in function <AckisRecipeList/List.lua:1794>
[string "@AckisRecipeList/List.lua"]:192: in function <AckisRecipeList/List.lua:191>

Locals:
(*temporary) = Ackis Recipe ListnormalFontObj {
 0 = <userdata>
}
(*temporary) = "Fonts\FRIZQT__.TTF"
(*temporary) = 10

**Comment by Sweepi** [comment]
Yeah with the Ulduar patch Blizzard bestowed the Dragonflight API changes on us classic Andys... will se what i can do.

**Comment by Xyrgia** [comment]
Interface\List.lua
1509?
fontObj:SetFont(font, fontSize)
fontObj:SetFont(font, fontSize, "OUTLINE")

1848?
acquire_tip:SetCell(1, 1, quality_color..recipe.name, "CENTER", 2)
acquire_tip:SetCell(1, 1, "|c"..quality_color..recipe.name.."|r", "CENTER", 2)

Scanner.lua
671/1139?
local targetID = tonumber(string.sub(UnitGUID("target"), -12, -7), 16)		-- Get the NPC ID
local _, _, _, _, _, targetID, _ = strsplit("-", UnitGUID("target"))		-- Get the NPC ID

**Comment by Xyrgia** [comment]
> Alchemy does the same. Info from bugsack: 772x AckisRecipeList/List.lua:1509: Usage: self:SetFont(fontFile, height, flags) [string "=[C]"]: in function `SetFont' [string "@AckisRecipeList/List.lua"]:1509: in function <AckisRecipeList/List.lua:1491> [string "@AckisRecipeList/List.lua"]:1880: in function <AckisRecipeList/List.lua:1794> [string "@AckisRecipeList/List.lua"]:192: in function <AckisRecipeList/List.lua:191>
> 
> Locals: (*temporary) = Ackis Recipe ListnormalFontObj { 0 = } (*temporary) = "Fonts\FRIZQT__.TTF" (*temporary) = 10

[see pull request](https://github.com/Sweepi/ARL_WotLK/pull/3)
